### PR TITLE
fix(nd): Remove ARC mask for TERR map from A380X ND

### PR DESCRIPTION
**Pull request by flogross89**
## Summary of Changes

- Keeps the round mask at the top of the ARC rose only for A32NX, not for A380X. Terrain mask is different on the A380

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Use new SimBridge version: https://www.swisstransfer.com/d/a0233bb1-53bc-438b-a0aa-bdf54f460a23

A32NX: 
Everything in the ARC TERR display should look as before, TERR should be cut off/masked at the top by the ARC ring

A380X:
Terrain mask should be square at the top when in ARC mode, and not cut off/masked by the ARC ring

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/systems/instruments/src/ND/style.scss [hunk 1]
```diff
@@ -236,3 +236,11 @@ path.rounded {
   stroke-linecap: round;
   stroke-linejoin: round;
 }
+
+.hidden {
+  display: none;
+}
+
+.a380xInvisible {
+  display: block;
```
Comment (tracernz) [inline fbw-a32nx/src/systems/instruments/src/ND/style.scss:245]: I don't think this should force block. Do we even need any style for it?

Comment (flogross89) [reply fbw-a32nx/src/systems/instruments/src/ND/style.scss:245]: empty rulesets are not allowed, or do you mean just not defining it in a32nx's style.scss? Don't know how that behaves. I would do `display: inherit` now

Comment (tracernz) [reply fbw-a32nx/src/systems/instruments/src/ND/style.scss]: That would also modify the behaviour. Why do we need a style in the A32NX CSS for this at all?

Comment (flogross89) [reply fbw-a32nx/src/systems/instruments/src/ND/style.scss]: Hmm yeah you're right. There's no a32nx/a380x prop passed down or defined anywhere, so I found this the best way to have an aircraft-specific style/visibility, but I'm open to other ideas. 
I could leave the style empty, although then lint complains

Comment (tracernz) [reply fbw-a32nx/src/systems/instruments/src/ND/style.scss]: You can just not have it in the a32nx stylesheet at all.

Comment (flogross89) [reply fbw-a32nx/src/systems/instruments/src/ND/style.scss]: Fixed that now

**Pull request merged**